docs/axes/cartesian/README.md

```diff
@@ -31,6 +31,7 @@ The following options are common to all cartesian axes but do not apply to other
 | `maxRotation` | `Number` | `90` | Maximum rotation for tick labels when rotating to condense labels. Note: Rotation doesn't occur until necessary. *Note: Only applicable to horizontal scales.*
 | `minRotation` | `Number` | `0` | Minimum rotation for tick labels. *Note: Only applicable to horizontal scales.*
 | `mirror` | `Boolean` | `false` | Flips tick labels around axis, displaying the labels inside the chart instead of outside. *Note: Only applicable to vertical scales.*
+| `offset` | `Boolean` | `false` | If true, extra space is added to the both edges and the axis is scaled to fit into the chart area. This is set to `true` in the bar chart by default.
```

Comment:
When `scale.bounds: 'ticks'` but the first and last data points are on the ticks, the scale won't add extra spaces at edges (see 5 below). However, `offset: true ` will always add them to make sure that the whole bar rectangles are displayed (see 7 below).

<img width="410" alt="screen shot 2017-08-06 at 10 34 06 am" src="https://user-images.githubusercontent.com/723188/29000071-e398257c-7a92-11e7-93bb-67b0365cc1e6.png">
<img width="400" alt="screen shot 2017-08-06 at 10 34 13 am" src="https://user-images.githubusercontent.com/723188/29000072-ef7d0b00-7a92-11e7-8ae5-e12d67b7aabd.png">

Internally, `offset: true ` doesn't change the time/pos lookup table. It just add one half size of the next tick intervals in `getPixelForValue()` (or `getPixelForOffset()` in case of a time scale), and this can be easily applied to other scales such as a category scale.